src/services/gitpod.ts

```diff
@@ -19,18 +21,21 @@ export const generateGitPodURL = (
         ? sourceCollections.length
         : sourceCollections.size;
 
-    const gitPodURL = `https://gitpod.io/#`;
+    // https://github.com/estuary/flow-gitpod-base/blob/main/init.sh
+    // This object is consumed by GitPod init.sh and has to stay in sync
+    const settingsPart = `FLOW_SETTINGS=${encodeURIComponent(
+        Buffer.from(
+            JSON.stringify({
+                id: draftId,
+                rt: token,
 
-    const urlVariables = `FLOW_DRAFT_ID=${encodeURIComponent(
-        draftId
-    )},FLOW_REFRESH_TOKEN=${encodeURIComponent(
-        Buffer.from(JSON.stringify(token)).toString('base64')
-    )},FLOW_TEMPLATE_TYPE=${derivationLanguage},FLOW_TEMPLATE_MODE=${
-        sourceCollectionCount > 1 ? 'multi' : 'single'
-    },FLOW_COLLECTION_NAME=${encodeURIComponent(computedEntityName)}`;
+                // TODO - (GitPod) These all seem unused in the script but need to double check
+                dl: derivationLanguage,
+                tt: sourceCollectionCount > 1 ? 'multi' : 'single',
+                cn: computedEntityName,
```

Comment:
Before merging I am going to test with these all being removed - but want to talk to back end first.